### ISSUE TITLE
Downloading files, filepath, filenames problem

### ISSUE THREAD

**Issue by aozen** [thedevdojo/voyager#3204]
- Laravel Version: 5.5.*
- Voyager Version: Last v.
- PHP Version: Last v.

### Description:
Hello awesome people, 

I have a question about downloading and pathing files from voyager admin panel. 

When i have download a file from panel. It names changing like this: 
original name = "1526552421example.txt"
but voyager saving like this = "_storage_1526552421example.txt" 
I don't get it why?
Copied and changed browse.blade like that. On row 155: 
`<a href="{{ Storage::disk('local')->url($data->{$row->field}) }}" target="_blank" download="{{ Storage::disk('local')->url($data->{$row->field}) }}">
Download
</a>`

This is working but i need to remove "_storage_". 
Also i can't get files from "storage/another_folder". I just download from storage folder with _storage_ name. I hope somebody can give me a idea. Thanks an advance.

**Comment by fletch3555** [comment]
Please show the slightest bit of effort and respect for us and provide the actual version numbers you're using.  Saying "last" or "latest" means absolutely nothing if this issue is looked at 6 months from now.

As for your issue, I have no idea what you're saying... Sounds to me like you're getting invalid names coming back, and you're trying to fix it by changing how the URL is generated.  I think your approach is WAY too focused.

If you're seeing actual filenames instead of GUIDs, then you've changed something from the Voyager defaults.  Providing your config and/or saying what was changed would be exceptionally beneficial for troubleshooting, as would actual code snippets (not one-liners with minimal or no context).

Voyager will store the uploaded filename as `[slug]\[month][year]\[filename].[ext]`([ref](https://github.com/the-control-group/voyager/blob/1.1/src/Http/Controllers/ContentTypes/File.php#L46)) as that's the file path in the configured storage/filesystem.  If you're seeing `storage`, you've likely changed something with the upload.

In short, you don't need to get rid of the `_storage_`, you need to figure out why it's changing it in the first place.  The default view should function perfectly fine for this, so having issues is a sign that you've done something else.

**Comment by emptynick** [comment]
Closing due to inactivity. Please provide more information.
